stats/bin/migrate.js

```diff
@@ -1,9 +1,28 @@
 import {
-  getPgPools,
   migrateEvaluateDB,
-  migrateStatsDB
+  migrateStatsDB,
+  getPgPools
 } from '@filecoin-station/spark-stats-db'
 
-const pgPools = await getPgPools()
-await migrateStatsDB(pgPools.stats)
-await migrateEvaluateDB(pgPools.evaluate)
+const {
+  DATABASE_URL,
+  EVALUATE_DB_URL
+} = process.env
+
+try {
+  console.log('Running migrations for stats database...')
+  const pgPools = await getPgPools()
+  
+  // @ts-ignore - PgPoolStats actually does have a query method at runtime
```

Comment:
Let's try to add correct types to migrate functions rather then using `@ts-ignore`. Using it should be avoided.